recipes/ibis-heavyai/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ibis-heavyai" %}
+{% set version = "1.0" %}
```

Comment:
Why are you releasing version 1.0 when version 2.0.5 was released in January? You can also get the source tar directly from GitHub instead of via PYPI; just swap the url and SHA.

Reply:
Because it's a new package in the end so it does not really matters. In the end, end users have to adjust things and a hard break is legitimate here.

As for where to get the package from, I followed the recipe template maker Grayskull. Why should we prefer GitHub over PyPI? Because to me PyPI seems to be the better way.

Reply:
If you keep your pypi releases up to date, there are no missing files from the pypi release, and there is no conflicting release information on GitHub, then there is no difference between getting the source from pypi or github.

However, the development URL for this package points here: https://github.com/heavyai/ibis-heavyai

This repository already has releases in version 2.0.x. I thought that maybe your PYPI releases are out of date which is why I suggested GitHub releases, but now I realize it's probably that you kept all of the git history from before the name change, so all the old tags and releases are still there.

If you start with version 1.0 under the new name you will eventually run into tag/release clashes in your GitHub repo which already has version 2.0.x and has already been forked by others and so every fork may contain these tags too. So I am suggesting that you continue versioning from 2.0.5 or start with 3.0.0 or clear out all those old tags, so there is no confusion.

Reply:
Thank you for pointing this out. I will keep this in mind.